sentry/info.json

```diff
@@ -10,7 +10,7 @@
         "sentry"
     ],
     "requirements": [
-        "sentry-sdk"
+        "sentry_sdk"

```

Comment:
[`sentry-sdk`](https://pypi.org/project/sentry-sdk/) is the correct name for the requirement. `sentry_sdk` just converts itself to `sentry-sdk` when you install with pip

I've tested the cog installation and there doesn't seem to be any issue with it installing the correct package and loading the cog

Reply:
Hmm, it wasn't installing correctly when I tested and this seemed to solve it 🤔 I'll do some more testing.

Reply:
Interesting. I'll try to replicate, but there *shouldn't* be an issue.
Have you tried adding the repo and installing the cog from scratch?

Reply:
So the main reason this started is me testing it on a redbot instance, and Red Not installing the package as described in the info json.